modules/contracts/src.ts/commitments/set-state-commitment.ts

```diff
@@ -111,13 +121,15 @@ export class SetStateCommitment implements EthereumCommitment {
       bnJson.versionNumber,
       bnJson.stateTimeout,
       bnJson.appIdentityHash,
+      bnJson.transactionData,
       bnJson.signatures[0],
       bnJson.signatures[1],
     );
   }
 
-  public async getSignedAppChallengeUpdate(): Promise<SignedAppChallengeUpdate> {
-    await this.assertSignatures();
+  public getSignedAppChallengeUpdate(): SignedAppChallengeUpdate {
+    // TODO: figure out where to do this rather than in this function
+    // await this.assertSignatures();
```

Comment:
Looks like we're calling the assertion in getSignedTransaction(), so is this TODO still relevant?